backend/uv-backend.py

```diff
@@ -0,0 +1,165 @@
+########################
+def check_existing_record(file_name, record_id):
+    file_object = u2py.File(file_name)
+    try:
+        record_object = file_object.read(record_id)
+        return True
+    except u2py.U2Error as e:
+        return False
+def mapping_customer(first_name,last_name,address,city,pfid):
+	details={}
+	details["firstName"]=first_name
+	details["lastName"]=last_name
+	details["address"]=address
+	details["city"]=city
+	if pfid!="":
+		details["pf"]="pf"
+	else:
+		details["pf"]=""
+	return details
+########################
+#### CUSTOMER API   ####
+########################
+@app.route('/api/customer', methods=['GET'])
+def customer_details():
+	customer_id=request.args.get('customerId')
+	customer_file= u2py.File("CUSTOMERS")
+	data=[]
+	cmd="LIST PHONE.NO F.NAME L.NAME ADDRESS CITY ZIP.CODE PHONE.LONG PFID DATA {} CUSTOMERS TOJSON".format(customer_id)
+	details=u2py.Command(cmd).run(capture=True)
+	details=json.loads(details)
+	del details['CUSTOMERS'][0]["_ID"]
+	values=details['CUSTOMERS'][0].values()
+	keys=["phoneNo","firstName","lastName","address","city","zipCode","altPhoneNo","pfid"]
+	customer_dict={key: value for key, value in zip(keys, values)}
+	data.append(customer_dict)
+	return Response(
+		json.dumps(data),
+		status=200,
+		mimetype='application/json')
+########################
+#### CONSULTANT API ####
+########################
+@app.route('/api/consultant',methods=['GET'])
+def consultant_details():
+	pdb.set_trace()
+	transaction_id=request.args.get('transactionId')
+	transaction_file=u2py.File("TRANSACTION")
+	trans_no=list(transaction_file.readv(transaction_id,1))[0][0]
+	phone_no=list(transaction_file.readv(transaction_id,2))[0][0]
+	em_file=u2py.File("EM")
+	consultant_details={}
+	data=[]
+	consultant_cmd="LIST FNAME LNAME SHORTNAME NICKNAME DATA {} EM TOJSON".format(phone_no)
+	details=u2py.Command(consultant_cmd).run(capture=True)
+	details=json.loads(details)
+	business_name=details['EM'][0]["NICKNAME"]
+	first_name=details['EM'][0]["FNAME"]
+	last_name=details['EM'][0]["LNAME"]
+	short_name=details['EM'][0]["SHORTNAME"]
+        ####checks the businessName if present it replaces the firstname
+	if(business_name!=''):
+		operator=str(business_name)
+	else:
+		operator=str(first_name)
+	operator=operator+" "+str(last_name)
+	if(len(operator)+len(short_name)<23):
+		operator=operator+" ("+str(short_name)+")"
+	consultant_details["operator"]=operator
+	####they only want length till 23(used for extra precaution)
+	operator=str(operator[0:23])
+	temp_type=list(transaction_file.readv(transaction_id,246))[0][0]
+	if temp_type =='':
+		temp_type="SLS CONSULT"
+	else:
+		temp_type=temp_type.split("*")
+		temp_type=str(temp_type[2])
+	employee_id=list(transaction_file.readv(transaction_id,244))[0][0]
+	if(check_existing_record("EM",employee_id)==True):
+	####sets name to Noconsultant if record not found
+		em_cmd="LIST FNAME LNAME SHORTNAME DATA {} EM TOJSON".format(employee_id)
```

Comment:
Avoid abbreviations, easy to type but hard to understand.
Instead of `em` go with `employee`.